src/main.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference path='./references.d.ts' />
```

Comment:
This needs to be imported or an @types package should be added to the project, e.g. `core-js` which can replace the `reflect-metadata` package as well. That would also mean that you wouldn't have to set the target to ES5 as far as I'm aware.